Sources/Fuzzilli/Core/CodeTemplate.swift

```diff
@@ -90,9 +93,12 @@ public class CodeTemplate {
                     methods.append(chooseUniform(from: fuzzer.environment.customMethodNames))
                 }
 
-                return .object(withProperties: properties, withMethods: methods)
+                if let group = group {
+                    return .object(ofGroup: group, withProperties: properties, withMethods: methods)
```

Comment:
Nit: I think the .object type constructor takes a String? as ofGroup parameter, so you should be able to remove the if-else and just leave line 97